app/assets/stylesheets/tabs.scss

```diff
@@ -0,0 +1,12 @@
+.content-box {
+  border: 1px solid red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.